build-system/common/check-package-manager.js

```diff
@@ -276,6 +276,14 @@ async function main() {
       yellow('\nWARNING: Detected problems with'),
       cyan(Array.from(updatesNeeded).join(', '))
     );
+    if (process.env.CI) {
+      console.log(
+        yellow('Skipping delay prompt for'),
+        cyan('CI'),
+        yellow('environment.')
+      );
+      return;
+    }
```

Comment:
The `node` Docker instance doesn't have Python installed, and the `openjdk` Docker container has a patch-version number lag - in both cases, these aren't issues (should we even be testing for the presence of Python in any of the commands except for the validator? and, should we even be testing for the LTS version of Node in the case of the validator tests?)

Reply:
> The `node` Docker instance doesn't have Python installed

This used to be a problem once upon a time because our non-validator devtools required python. E.g. The nailgun closure compiler. Not a requirement these days.

> and the `openjdk` Docker container has a patch-version number lag

Validator tests directly invoke `python3`, so if that's available, we're good.

https://github.com/ampproject/amphtml/blob/0fde716d914c98d44ca41d93c1f2469b3a61c87e/build-system/tasks/validator.js#L30-L35
https://github.com/ampproject/amphtml/blob/0fde716d914c98d44ca41d93c1f2469b3a61c87e/build-system/tasks/validator.js#L64-L69

> should we even be testing for the LTS version of Node in the case of the validator tests?

> In any case, the delay doesn't really do anything in the CI

The `check-package-manager.js` script is meant to ensure that the environment is properly set up for local development. It has occasionally proved useful to debug issues during CI, so we still run it. But you've convinced me that the 10 second delay is useless during CI 😃 

My only suggestion: maybe simplify this block to return during CI without printing the "skipping delay prompt" message?

Reply:
I want to keep it exactly because it's the kind of helpful debugging statement that you just mentioned - if it shows up and we are experiencing deps issues, this would be a hint to look at

Reply:
SGTM.